docker-services.yml

```diff
@@ -82,8 +82,10 @@ services:
     image: docker.elastic.co/elasticsearch/elasticsearch-oss:6.6.0
```

Comment:
Are you sure you change the elasticsearch version to 7? Or 6.6?

Reply:
Ups, I changed it to version 7.